cmd/kubeadm/app/cmd/init.go

```diff
@@ -155,6 +156,10 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 		"Port for JWS discovery service to bind to",
 	)
 
+	cmd.PersistentFlags().StringVar(
+		&cfg.ImagePrefix, "image-prefix", gcrPrefix,
+		"use specific images instead of gcr",
```

Comment:
A command line flag is more friendly for kubeadm users. 
In fact, I have a more radical branch( [pre pull images and configurable pod](https://github.com/bulletRush/kubernetes/commit/7d73b13bd4a4b346dd13fd0d6808a3a49e37110a) ) for personal dev and test.